nautobot/core/api/views.py

```diff
@@ -422,8 +420,6 @@ class StatusView(NautobotAPIVersionMixin, APIView):
                     "nautobot-version": {"type": "string"},
                     "plugins": {"type": "object"},
                     "python-version": {"type": "string"},
-                    # 2.0 TODO: remove rq-workers-running property
-                    "rq-workers-running": {"type": "integer"},
```

Comment:
Should this API change be documented in `docs/installation/upgrading-from-nautobot-v1.md`